### ISSUE TITLE
Three-person thread not working

### ISSUE THREAD

**Issue by snan** [deltachat/deltachat-ios#1437]
I'm in a three person thread/group, the other two people are on normal MUA. One is on Hotmail. His responses to the group (from Hotmail) don't show up in the group in my Delta Chat. But looking at the messages in my other MUA, and on my imap server, he has both of our addresses in the `To:`-field like God (and RFC 2822) intended.

**Comment by hpk42** [comment]
i suspect it's not easy to fix it (and rather in the core library anyway).  Could you provide an example source code of his mail that is not correctly associated to your chat (you can obfuscaste the subject and content but please leave the other headers intact)?

**Comment by hpk42** [comment]
On Fri, Jan 14, 2022 at 09:02 -0800, snan wrote:
> Of course, thank you ♥
> https://ellen.idiomdrottning.org/delta-weird-mail.txt
> Also, I misremembered Hotmail and Gmail (I use neither)

thanks! So you are using DC and the other two (friend1/friend2) are not, right? 
You created the group and if they reply to a message from you then it arrives in your chat correctly, right? 

the mail you posted is probably rather about friend1 replying to friend2, so not directly a reply to a message from you.  This should still work, though.  

Btw, you forgot to state the version of Delta Chat you are using. I assume it's the currently latest from apple store? (1.26 i think)

**Comment by snan** [comment]
> You created the group and if they reply to a message from you then it arrives in your chat correctly, right?

That is incorrect. I have some messages from him replying to me, with both me and third friend in the To: header, showing up in his chat in Delta Chat and not in the group, while other times it does show up in the group.

That's right that I'm the only one using Delta Chat, 1.26.2.
